spec/requests/pages_spec.rb

```diff
@@ -36,6 +36,8 @@
     it "has proper headline" do
       get "/privacy"
       expect(response.body).to include("Privacy Policy")
+      expect(response.body).to include("https://shop.dev.to")
+      expect(response.body).to include("DEV(local) Shop")
```

Comment:
I would think this should be `include ("#{ApplicationConfig['COMMUNITY_NAME']} Shop")`